home.css

```diff
@@ -1,6 +1,10 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
 @import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css");
 
+html {
+  scroll-behavior: smooth;
+}
+
 body {
```

Comment:
There's a horizontal scroll in both versions that makes for bad UI/UX. I'm using Google Chrome for reference.

<img width="228" alt="Screen Shot 2022-07-27 at 9 02 56" src="https://user-images.githubusercontent.com/82426682/181266906-a1eb6725-71b9-4a71-9526-64ef2f10f92d.png">

Please fix it.